build.cmd won't run on Windows with only .NET Core installed

```
Downloading NuGet.exe ...
Feeds used:
  C:\Users\Nikos\.nuget\packages\
  https://api.nuget.org/v3/index.json



Attempting to gather dependency information for package 'FAKE.Core.4.63.2' with respect to project 'build\tools', targeting 'Any,Version=v0.0'
Gathering dependency information took 1.65 sec
Attempting to resolve dependencies for package 'FAKE.Core.4.63.2' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'FAKE.Core.4.63.2'
Resolved actions to install package 'FAKE.Core.4.63.2'
Retrieving package 'FAKE.Core 4.63.2' from 'C:\Users\Nikos\.nuget\packages\'.
Adding package 'FAKE.Core.4.63.2' to folder 'C:\Snapshot\Fare\build\tools'
Added package 'FAKE.Core.4.63.2' to folder 'C:\Snapshot\Fare\build\tools'
Successfully installed 'FAKE.Core 4.63.2' to build\tools
Executing nuget actions took 436.42 ms
Running FAKE Build...
C:\Program Files\Git\mingw64\bin\git.exe describe --tags --long --abbrev=40 --first-parent --match=v*
v2.1.1-1-g71eaf12a6553a6ddd84a0fc52e78f19ada766d0a
Building project with version: LocalBuild
Shortened DependencyGraph for Target CompleteBuild:
<== CompleteBuild
   <== NuGetPack
      <== CleanNuGetPackages
      <== Test
         <== Build
            <== Verify
               <== RestoreNuGetPackages
               <== VerifyOnly
            <== BuildOnly
         <== TestOnly

The running order is:
  - CleanNuGetPackages
  - RestoreNuGetPackages
  - VerifyOnly
  - Verify
  - BuildOnly
  - Build
  - TestOnly
  - Test
  - NuGetPack
  - CompleteBuild
Running build with 1 worker
Starting Target: CleanNuGetPackages
Creating C:\Snapshot\Fare\build\NuGetPackages
Finished Target: CleanNuGetPackages
Starting Target: RestoreNuGetPackages
Building project: Src/Fare.sln
  c:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe  Src/Fare.sln /t:Restore /m  /nodeReuse:False  /v:m  /p:RestorePackages="False" /p:AssemblyVersion="2.1.0.0" /p:FileVersion="2.1.1.0" /p:InformationalVersion="2.1.1.1-71eaf12a6553a6ddd84a0fc52e78f19ada766d0a" /p:PackageVersion="2.1.1.1" /logger:Fake.MsBuildLogger+ErrorLogger,"C:\Snapshot\Fare\build\tools\FAKE.Core\tools\FakeLib.dll"
Microsoft (R) Build Engine version 4.7.3056.0
[Microsoft .NET Framework, version 4.0.30319.42000]
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Snapshot\Fare\Src\Fare.sln.metaproj : error MSB4057: The target "Restore" does not exist in the project. [C:\Snapshot\Fare\Src\Fare.sln]
Running build failed.
Error:
Building Src/Fare.sln failed with exitcode 1.

---------------------------------------------------------------------
Build Time Report
---------------------------------------------------------------------
Target                 Duration
------                 --------
CleanNuGetPackages     00:00:00.0025071
RestoreNuGetPackages   Failure
Total:                 00:00:01.0394329
---------------------------------------------------------------------
Status:                Failure
---------------------------------------------------------------------
---------------------------------------------------------------------
  1) Building Src/Fare.sln failed with exitcode 1.
  2) MSB4057: C:\Snapshot\Fare\Src\Fare.sln.metaproj(0,0): The target "Restore" does not exist in the project.
---------------------------------------------------------------------
```

#30, #31, /cc @zvirja

--- a/Build.fsx
+++ b/Build.fsx
@@ -79,47 +79,26 @@ let buildVersion = match getBuildParamOrDefault "BuildVersion" "git" with
                                       infoVersion = getBuildParamOrDefault "BuildInfoVersion" assemblyVer
                                       nugetVersion = getBuildParamOrDefault "BuildNugetVersion" assemblyVer
                                       source = None }
+                                      
+let runDotNet command configuration properties =
+    let stringProps = properties
+                      @ [ "AssemblyVersion", buildVersion.assemblyVersion
+                          "FileVersion", buildVersion.fileVersion
+                          "InformationalVersion", buildVersion.infoVersion
+                          "PackageVersion", buildVersion.nugetVersion ]
+                      |> Seq.map(fun (name, value) -> sprintf "/p:%s=%s" name value )
+                      |> String.concat " "
+                      
+    DotNetCli.RunCommand id
+                         (sprintf "%s %s --configuration %s %s" command solutionToBuild configuration stringProps)
 
-let runMsBuild target configuration properties =
-    let verbosity = match getBuildParam "BuildVerbosity" |> toLower with
-                    | "quiet" | "q"         -> Quiet
-                    | "minimal" | "m"       -> Minimal
-                    | "normal" | "n"        -> Normal
-                    | "detailed" | "d"      -> Detailed
-                    | "diagnostic" | "diag" -> Diagnostic
-                    | _ -> Minimal
+Target "Verify" (fun _ -> runDotNet "build" configuration [])
+Target "Build" (fun _ -> runDotNet "build" configuration [])
 
-    let configProperty = match configuration with
-                         | Some c -> [ "Configuration", c ]
-                         | _ -> []
-
-    let properties = configProperty @ properties
-                     @ [ "AssemblyVersion", buildVersion.assemblyVersion
-                         "FileVersion", buildVersion.fileVersion
-                         "InformationalVersion", buildVersion.infoVersion
-                         "PackageVersion", buildVersion.nugetVersion ]
-
-    solutionToBuild
-    |> build (fun p -> { p with MaxCpuCount = Some None
-                                Verbosity = Some verbosity
-                                Targets = [ target ]
-                                Properties = properties })
-
-let rebuild configuration = runMsBuild "Rebuild" (Some configuration) []
-
-Target "RestoreNuGetPackages" (fun _ -> runMsBuild "Restore" None [])
-
-Target "VerifyOnly" (fun _ -> rebuild "Verify")
-
-Target "BuildOnly" (fun _ -> rebuild configuration)
-Target "TestOnly" (fun _ ->
+Target "Test" (fun _ ->
     DotNetCli.RunCommand (fun p -> { p with WorkingDir = testProjectDir })
-                         (sprintf "xunit --configuration %s --no-build -x86" configuration)
+                         (sprintf "test --configuration %s" configuration)
 )
-
-Target "Verify" DoNothing
-Target "Build"  DoNothing
-Target "Test"   DoNothing
 
 Target "CleanNuGetPackages" (fun _ ->
     CleanDir nuGetOutputFolder
@@ -127,11 +106,9 @@ Target "CleanNuGetPackages" (fun _ ->
 
 Target "NuGetPack" (fun _ ->
     // Pack projects using MSBuild.
-    runMsBuild "Pack" (Some configuration) [ "IncludeSource", "true"
-                                             "IncludeSymbols", "true"
-                                             "PackageOutputPath", FullName nuGetOutputFolder
-                                             "NoBuild", "true" ]
-
+    runDotNet "pack" configuration [ "IncludeSource", "true"
+                                     "IncludeSymbols", "true"
+                                     "PackageOutputPath", FullName nuGetOutputFolder ]
 )
 
 let publishPackagesWithSymbols packageFeed symbolFeed accessKey =
@@ -159,18 +136,8 @@ Target "PublishNuGetPublic" (fun _ ->
 
 Target "CompleteBuild"   DoNothing
 
-"RestoreNuGetPackages" ==> "Verify"
-"VerifyOnly"           ==> "Verify"
-
-"Verify"    ==> "Build"
-"BuildOnly" ==> "Build"
-
-"BuildOnly" ==> "TestOnly"
-
-"Build"    ==> "Test"
-"TestOnly" ==> "Test"
-
 "CleanNuGetPackages" ==> "NuGetPack"
+"Verify"             ==> "NuGetPack"
 "Test"               ==> "NuGetPack"
 
 "NuGetPack" ==> "CompleteBuild"

--- a/Src/Fare.IntegrationTests/Fare.IntegrationTests.csproj
+++ b/Src/Fare.IntegrationTests/Fare.IntegrationTests.csproj
@@ -30,7 +30,6 @@
 
     <PackageReference Include="xunit" Version="2.3.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0" />
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.0" />
 
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
   </ItemGroup>

--- a/Src/Fare.sln
+++ b/Src/Fare.sln
@@ -7,6 +7,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Fare", "Fare\Fare.csproj", 
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Fare.IntegrationTests", "Fare.IntegrationTests\Fare.IntegrationTests.csproj", "{20F5CC3B-9D71-472A-BD28-3A4C104A00E4}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{84A69DF5-281E-45DC-817F-663008351957}"
+	ProjectSection(SolutionItems) = preProject
+		..\.editorconfig = ..\.editorconfig
+		..\appveyor.yml = ..\appveyor.yml
+		..\Build.fsx = ..\Build.fsx
+	EndProjectSection
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU

